R/html.R

```diff
@@ -847,7 +847,7 @@ parse_footnotes = function(x) {
   j = min(j[j > i])
   n = length(x)
   r = '<li id="fn([0-9]+)"><p>.+?<a href="#fnref\\1"[^>]*?>.</a></p></li>'
-  s = paste(x[i:n], collapse = '')
+  s = paste(x[i:n], collapse = ' ')
```

Comment:
I guess `\n` is better than a space here.

Reply:
Done.